src/test/java/bio/terra/service/dataset/DatasetServiceUnitTest.java

```diff
@@ -503,4 +514,34 @@ void createDataset(boolean isTDRBillingProfile) {
         flightMap.get(JobMapKeys.TDR_BILLING_PROFILE_FALLBACK.getKeyName(), Boolean.class),
         equalTo(isTDRBillingProfile));
   }
+
+  @ParameterizedTest
+  @MethodSource("provideIamRoleName")
+  void testIsInherited(String role, boolean isInherited) {
+    assertThat(datasetService.isInheritedRole(role), is(isInherited));
```

Comment:
I think these uses should use the class, not an instance
```suggestion
    assertThat(DatasetService.isInheritedRole(role), is(isInherited));
```